domains/authorization/Proxy.IntegrationTests/V1EndpointTests.cs

```diff
@@ -0,0 +1,225 @@
+            new object[] {"/v1/transfers", new TransferRequest{ Quantity = 1, CertificateId = new FederatedStreamId(){ Registry = "test", StreamId = Guid.NewGuid()}, ReceiverId = Guid.NewGuid(), HashedAttributes = new []{"None :D"}}},
+        };
+
+        public IEnumerator<object[]> GetEnumerator() => _data.GetEnumerator();
+
+        IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
+    }
+
+    [Theory]
+    [ClassData(typeof(V1PostTestData))]
+    public async Task GivenB2C_WhenV1PostEndpointsAreUsed_WithoutEoApiVersion_Return200Ok(string endpoint, object requestBody)
+    {
+        // Arrange
+        var requestBuilder = Request.Create()
+            .WithPath($"{endpoint}")
+            .WithBody(JsonSerializer.Serialize(requestBody))
+            .UsingPost();
+
+        fixture.WalletWireMockServer
+            .Given(requestBuilder)
+            .RespondWith(
+                Response.Create()
+                    .WithStatusCode(200)
+                    .WithHeader("Content-Type", "text/plain")
+                    .WithBody("{{request.headers.wallet-owner}}")
+                    .WithTransformer()
+            );
+
+        var orgId = Guid.NewGuid().ToString();
+        var client = CreateClientWithOrgAsSub(orgId);
+
+        // Act
+        var response = await client.PostAsync($"{endpoint}", new StringContent(JsonSerializer.Serialize(requestBody), Encoding.UTF8, "application/json"));
+        var responseContent = await response.Content.ReadAsStringAsync();
+
+        // Assert
+        response.StatusCode.Should().Be(HttpStatusCode.OK);
+        responseContent.Should().Be(orgId);
+    }
+
+    public class V20250101PostTestData : IEnumerable<object[]>
+    {
+        private readonly List<object[]> _data = new List<object[]>
+        {
+            new object[] {"/wallets", new CreateWalletRequest{ PrivateKey = Encoding.ASCII.GetBytes("test") }},
+            new object[] {"/wallets/8229a340-1c9d-46b6-8212-b767e42e02f0/endpoints", new {}},
+            new object[] {"/external-endpoints", new CreateExternalEndpointRequest{TextReference = "Hello", WalletReference = new WalletEndpointReference(){ Endpoint = new Uri("https://test"), Version = 0, PublicKey = "test"}}},
+            new object[] {"/claims", new ClaimRequest{ Quantity = 1, ConsumptionCertificateId = new FederatedStreamId(){ Registry = "test", StreamId = Guid.NewGuid()}, ProductionCertificateId = new FederatedStreamId(){ Registry = "test", StreamId = Guid.NewGuid()}}},
+            // slices is our only insecure endpoint. new object[] {"/slices", new ReceiveRequest{ Quantity = 1, CertificateId = new FederatedStreamId(){ Registry = "test", StreamId = Guid.NewGuid()}, PublicKey = Encoding.ASCII.GetBytes("test"), RandomR = Encoding.ASCII.GetBytes("test"), Position = 1, HashedAttributes = new List<HashedAttribute>()}},
```

Comment:
What is this? 🤔

Reply:
Slices does not have authentication and the test fails because we don't have to use Header on calls to it.

Reply:
Please do not resolve comments, unless specifically asked, next time ;)